### ISSUE TITLE
Gun safes should be welded to the floor or require a do-after to move them

### ISSUE THREAD

**Issue by Kayek14** [space-wizards/space-station-14#23389]
## Description

--Feature request--

Gun safes ironically make stealing guns easier since you can just kinda drag them out. What use are they being locked when they are not secured to the floor? 

Just obseved a nukie round on a station where the armory is next to space. One C4 to breach into the armory and three nukies were able to 100% clean out the armory in under 10 seconds because all the guns were locked in safes.

I propose that gun safes should be welded to the floor and require a do-after to be able to move them.

**Comment by IProduceWidgets** [comment]
Hmm. Should they still be able to be unwelded and dragged around, or should they just have their sharedpullable component yoinked? 

I'm kind of leaning toward the latter because otherwise it's only slightly more inconvenient.

**Comment by LankLTE** [comment]
Couldn't you, as an alternative, give them a really high mass so they're a pain in the ass to drag anywhere? That could help in addition to unwelding, but removing their ability to be pulled outright is probably not ideal.

**Comment by K-Dynamic** [comment]
Increase their mass, add another wall to armoury + mines + sec turret

Make nukies work to bust up armoury

**Comment by deltanedas** [comment]
anchorable but specify welding tool quality :trollface:

**Comment by K-Dynamic** [comment]
Yeah but nukies will just spend an extra second unwelding the gun lockers before tossing them into space

**Comment by deltanedas** [comment]
and long doafter :trollface:

**Comment by Hmeister-real** [comment]
Make the armory safes have a key to open rather than access (In the wardens office and 2 in the HoS office or smthn) and make them more durable and explosion proof (mostly). Thats a solution
